app/models/manageiq/providers/base_manager/event_catcher/runner.rb

```diff
@@ -95,16 +95,16 @@ def before_exit(message, _exit_code)
   def sync_blacklisted_events
```

Comment:
I think this can go away entirely - we already sync_settings

Reply:
Yes I'm testing that the `@filtered_events` ivar is also reloaded when settings change (ref vs copy) when pointing to the Config object

Reply:
Yup confirmed that `@filtered_events` had the old copy of the array but after `sync_config` `@ems.class.default_blacklisted_event_names` has the updated list so that's perfect.